src/modules/import/domain/import-event-handler.ts

```diff
@@ -0,0 +1,162 @@
+import { DomainError } from '../../../shared/error/domain.error.js';
+import { EntityNotFoundError } from '../../../shared/error/entity-not-found.error.js';
+import { OrganisationRepository } from '../../organisation/persistence/organisation.repository.js';
+import { Organisation } from '../../organisation/domain/organisation.js';
+import { ImportDataRepository } from '../persistence/import-data.repository.js';
+import {
+    PersonenkontextCreationService,
+    PersonPersonenkontext,
+} from '../../personenkontext/domain/personenkontext-creation.service.js';
+import { DbiamCreatePersonenkontextBodyParams } from '../../personenkontext/api/param/dbiam-create-personenkontext.body.params.js';
+import { OrganisationsTyp } from '../../organisation/domain/organisation.enums.js';
+import { ImportDataItem } from './import-data-item.js';
+import { ImportVorgang } from './import-vorgang.js';
+import { ImportVorgangRepository } from '../persistence/import-vorgang.repository.js';
+import { ImportExecutedEvent } from '../../../shared/events/import-executed.event.js';
+import { EventHandler } from '../../../core/eventbus/decorators/event-handler.decorator.js';
+import { OrganisationByIdAndName } from './import-workflow.js';
+import { Injectable } from '@nestjs/common';
+import { ClassLogger } from '../../../core/logging/class-logger.js';
+import { ImportPasswordEncryptor } from './import-password-encryptor.js';
+@Injectable()
+export class ImportEventHandler {
```

Comment:
Manche Kommentare sind Deutsch andere wiederum Englisch. Insgesamt finde ich auch es sind zu viele Kommentare im Code. Manches sollte man besser in ein Folgeticket schreiben.

Reply:
Es ist nur Refactoring, die Logik, die gab es schon. Ich lösche die 👍